src/api/crypto.js

```diff
@@ -128,14 +130,19 @@ module.exports.pubSignKeyFromRaw = (appHandle, raw) => {
 };
 
 /**
- * Interpret the secret SignKey from a given raw string
+ * Interpret the secret SignKey from a given raw buffer
  * @name window.safeCrypto.secSignKeyFromRaw
  *
  * @param {SAFEAppHandle} appHandle the app handle
  * @param {(String|Buffer)} raw the raw input string
  *
- * @returns {Promise<SecSignKeyHandle>} 
+ * @returns {Promise<SecSignKeyHandle>}
  *
+ * @example // Interpret the secret sign key from raw buffer
+ * const signKeyPairHandle = await window.safeCrypto.generateSignKeyPair(appHandle);
+ * let secretSignKeyHandle = await window.safeaCryptoSignKeyPair.getSecSignKey(signKeyPairHandle);
+ * const rawSecretSignKey = await window.safeCryptoSecSignKey.getRaw(secretSignKeyHandle);
+ * secretSignKeyHandle = await window.safeCrypto.secSignKeyFromRaw(appHandle, rawSecretKey);
```

Comment:
I think this should be `secretSignKeyHandle = await window.safeCrypto.secSignKeyFromRaw(appHandle, rawSecretSignKey);`.
We'd need to translate all the example to use `await` to be consistent, we can do this separately.